doc/ref/lists.xml

```diff
@@ -1815,8 +1815,8 @@ An <E>enumerator</E> is an immutable list that need not store its elements
 explicitly but knows, from a set of basic data,
 how to determine the <M>i</M>-th element and the position of a given object.
 A typical example of this is a vector space over a finite field with <M>q</M>
-elements, say, for which it is very easy to enumerate all elements
-using <M>q</M>-adic expansions of integers.
+elements, where <M>q</M> is a prime, for which it is very easy to
```

Comment:
Why is `q` limited to a prime here, as opposed to being a non-trivial prime power? Or is that an accident? If so, instead of fixing that insertion, perhaps just undo it? Because I'd expect most if not all readers to know that the size of a finite field always is a prime power, and even if not, pointing that out is not really relevant for the example, is it? After all, q-adic expansion is also define for q which is not a prime power.

Reply:
Maybe this is completely ignorant of me, but I thought `p`-adic extension is defined for prime numbers in the standard literature? @wilfwilson pointed this out to me above. Originally, I put `where <M>q</M> is a prime power`. Of course, a field can have prime-power order, but my impression was that `q`-adic expansion always refers to a prime number.

Reply:
g-adic expansion is defined for any integer g >= is 2; for g=10 you use it every day to write numbers ... :-) See also the GAP function `CoefficientsQadic`.

Reply:
Oh my.... Can I please delete this thread? 🤦‍♂

Reply:
Don’t worry @DominikBernhardt - it’s fine for us to be ignorant of parts of mathematics and terminology (it’s a big subject 😉).